apps/nrm/views/types-of-exploitation.html

```diff
@@ -1,49 +1,49 @@
+        {{#t}}fields.types-of-exploitation-forced-to-work.hint{{/t}}
     </span>
     {{#renderField}}types-of-exploitation-wages-taken{{/renderField}}
     {{#renderField}}types-of-exploitation-forced-to-commit-fraud{{/renderField}}
     <hr/>
-    <h3 class="govuk-heading-m">Sexual exploitation</h3>
+    <h3 class="govuk-heading-m">{{#t}}pages.types-of-exploitation.heading-2{{/t}}</h3>
     {{#renderField}}types-of-exploitation-prostitution{{/renderField}}
     {{#renderField}}types-of-exploitation-child-exploitation{{/renderField}}
     <span class="govuk-hint govuk-checkboxes__hint checkbox-hint">
-      This includes child sex trafficking.
+        {{#t}}fields.types-of-exploitation-child-exploitation.hint{{/t}}
     </span>
     {{#renderField}}types-of-exploitation-taken-somewhere{{/renderField}}
     <span class="govuk-hint govuk-checkboxes__hint checkbox-hint">
-      This is different to rape.
+        {{#t}}fields.types-of-exploitation-child-exploitation.hint{{/t}}
     </span>
     <hr/>
-    <h3 class="govuk-heading-m">Criminal exploitation</h3>
+    <h3 class="govuk-heading-m">{{#t}}pages.types-of-exploitation.heading-3{{/t}}</h3>
     {{#renderField}}types-of-exploitation-forced-to-commit-crime{{/renderField}}
     <span class="govuk-hint govuk-checkboxes__hint checkbox-hint">
-      This includes being used as a drug mule across county lines. Victims will often be unpaid although they may get food and somewhere to live.
+        {{#t}}fields.types-of-exploitation-forced-to-commit-crime.hint{{/t}}
     </span>
     {{#renderField}}types-of-exploitation-organs-removed{{/renderField}}
     <hr/>
-    <h3 class="govuk-heading-m">Domestic servitude</h3>
+    <h3 class="govuk-heading-m">{{#t}}pages.types-of-exploitation.heading-4{{/t}}</h3>
     {{#renderField}}types-of-exploitation-unpaid-household-work{{/renderField}}
     <span class="govuk-hint govuk-checkboxes__hint checkbox-hint">
-      Victims who are exploited by relatives may have been forced into an arranged marriage and abused in other ways.
+        {{#t}}fields.types-of-exploitation-unpaid-household-work.hint{{/t}}
     </span>
     <hr/>
     {{#renderField}}types-of-exploitation-other{{/renderField}}
     <hr />
     <div class="govuk-warning-text">
       <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
       <strong class="govuk-warning-text__text">
-        <span class="govuk-warning-text__assistive">Warning</span>
-         If you think they are a victim of another crime you can report it to the police by calling 101.
+        <span class="govuk-warning-text__assistive">{{#t}}pages.types-of-exploitation.warning-heading{{/t}}</span>
+        {{#t}}pages.types-of-exploitation.warning{{/t}}
       </strong>
     </div>
     {{#input-submit}}continue{{/input-submit}}
   {{/page-content}}
-{{/partials-page}}
+{{/partials-page}}
```

Comment:
EOF line?

Reply:
Actioned in PR regarding MS-734 Do they want their case referred to NRM?